students/BrandonHenson/Lesson9/Mailroom_5.py

```diff
@@ -0,0 +1,151 @@
+# Brandon Henson
+# 5/20/18
+# Lesson 09 mailroom_5.py
+# Fixing issues:Refactoring
+# !/usr/bin/env python3
+
+
+class Donor():
+    def __init__(self, name=None):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, val):
+        self._name = val
```

Comment:
you don't need to use properties if you aren't doing anything special to the attributes (like we did for `area` in Circle assignment) or you have to protect access to the attribute (here single underscore will not protect the attribute); as it is here it provides no value.